pkg/ui/src/views/cluster/util/graphs.ts

```diff
@@ -63,7 +63,10 @@ class AxisDomain {
     const max = extent[1];
     if (alignMinMax) {
       const alignedMin = min - min % increment;
-      const alignedMax = max - max % increment + increment;
+      let alignedMax = max;
```

Comment:
This change looks like it could have a much wider ranging impact than that described by the commit message.  Can you provide examples of the change or non-change that occurs to other, non-percentage charts?